rest/taskana-rest-spring/src/test/java/pro/taskana/common/rest/TestSchemaNameCustomizable.java

```diff
@@ -8,7 +8,6 @@
 import java.sql.SQLException;
```

Comment:
@gitgoodjhe what the hell is this class? Can we please talk about this?

- resetDb() is not annotated with `@BeforeEach`
- class name does not end with `Test` and thus this class will not be found by surefire.
- general setup is questionable
- test content is too complicated.. 

Like what the hell?!

Reply:
First time I ever saw this class :)